Entities/ADoor.h

```diff
@@ -96,6 +96,54 @@ namespace RTE {
 		/// </summary>
 		/// <returns>Whether a player can control this at all.</returns>
 		bool IsControllable() const override { return false; }
+
+		/// <summary>
+		/// Gets this ADoor's Door Move Start Sound. Ownership is not transferred!
+		/// </summary>
+		/// <returns>The SoundContainer for this ADoor's Door Move Start Sound.</returns>
+		SoundContainer* GetDoorMoveStartSound() const { return m_DoorMoveStartSound; }
+
+		/// <summary>
+		/// Sets this ADoor's Door Move Start Sound.
+		/// </summary>
+		/// <param name="newSound">The new SoundContainer for this ADoor's Door Move Start Sound</param>
```

Comment:
See comments in ACrab.h about `*` placement, method comment capitalization and `.`s at the end of method comments.